scripts/publish/publish.sh

```diff
@@ -167,13 +169,24 @@ upload_installers_to_blob_storage(){
         return 1
     fi
 
-    # create the latest blob
-    echo "Updating the latest dotnet installer.."
-    local extension="${filename##*.}"
-    local latestblob="$CHANNEL/Installers/Latest/dotnet-$OSNAME-x64.latest.$extension"
+    if $makelatest; then
+        # create the latest blob
+        echo "Updating the latest dotnet installer.."
+        local extension="${filename##*.}"
+        local latestblob="$CHANNEL/Installers/Latest/dotnet-$OSNAME-x64.latest.$extension"
 
-    if ! upload_file_to_blob_storage_azure_cli $latestblob $installfile; then
-        return 1
+        if ! upload_file_to_blob_storage_azure_cli $latestblob $installfile; then
+            return 1
+        fi
+
+        local localVersionFilename="latest.$OSNAME.version"
+        local latestVersionBlob="$CHANNEL/Installers/Latest/$localVersionFilename"
+        rm -f "$SCRIPT_DIR/$localVersionFilename"
+        echo "$(_get_pkg_version $filename)" > "$SCRIPT_DIR/$localVersionFilename"
```

Comment:
Also I would like to keep the same logic for unix and windows. Diverging it may lead to issues later. So can you also make sure your changes make it to publish.ps1?